en/micro-integrator/docs/references/connectors/ISO8583-connector/ISO8583-inbound-endpoint-example.md

```diff
@@ -0,0 +1,106 @@
+
+3. Click on **Inbound Endpoint** in design view and under `properties` tab, update class name to `org.wso2.carbon.inbound.iso8583.listening.ISO8583MessageConsumer`. 
+
+4. Navigate to source view and update it with following config as required. 
+
+   ```xml
+   <?xml version="1.0" encoding="UTF-8"?><inboundEndpoint xmlns="http://ws.apache.org/ns/synapse" name="custom_listener" sequence="requestISO" onError="fault" class="org.wso2.carbon.inbound.iso8583.listening.ISO8583MessageConsumer" suspend="false">
+       <parameters>
+           <parameter name="inbound.behavior">listening</parameter>
+           <parameter name="sequential">true</parameter>
+           <parameter name="coordination">true</parameter>
+           <parameter name="port">5000</parameter>
+           <parameter name="headerLength">2</parameter>
+           <parameter name="isProxy">false</parameter>
+       </parameters>
+   </inboundEndpoint>
+   ```
+   Sequence to process the message.
+   
+   In this example for simplicity we will just log the message, but in a real world use case, this can be any type of message mediation.
+ 
+   ```xml
+   <?xml version="1.0" encoding="UTF-8"?><sequence xmlns="http://ws.apache.org/ns/synapse" name="requestISO" onError="fault">
+       <log level="full">
+           <property name="Log_Message for ISO8583 Inbound Endpoint" value="Message received from sample1-source"/>
+       </log>
+   </sequence>
+   ```
+## Exporting Integration Logic as a CApp
+
+**CApp (Carbon Application)** is the deployable artefact on the Enterprise Integrator runtime. Let us see how we can export integration logic we developed into a CApp. To export the `Solution Project` as a CApp, a `Composite Application Project` needs to be created. Usually, when a solution project is created, this project is automatically created by Integration Studio. If not, you can specifically create it by navigating to  **File** -> **New** -> **Other** -> **WSO2** -> **Distribution** -> **Composite Application Project**. 
+
+1. Right click on Composite Application Project and click on **Export Composite Application Project**.</br> 
+  <img src="/assets/img/connectors/capp-project1.png" title="Export as a Carbon Application" width="300" alt="Export as a Carbon Application" />
+
+2. Select an **Export Destination** where you want to save the .car file. 
+
+3. In the next **Create a deployable CAR file** screen, select inbound endpoint and sequence artifacts and click **Finish**. The CApp will get created at the specified location provided at the previous step. 
+
+## Deployment
+
+1. Navigate to [connector store](https://store.wso2.com/store/assets/esbconnector/list) and search for `ISO8583`. Click on `ISO8583 Inbound Endpoint` and download the jar file by clicking on `Download Inbound Endpoint`. Copy this jar file into  <PRODUCT-HOME>/lib folder. 
+
+2. Download `jpos-1.9.4.jar`, `jdom-1.1.3.jar`, `commons-cli-1.3.1.jar` and add it to <PRODUCT-HOME>/lib folder.
```

Comment:
```suggestion
2. Download `jpos-1.9.4.jar`, `jdom-1.1.3.jar`, and `commons-cli-1.3.1.jar` and add it to <PRODUCT-HOME>/lib folder.
```
From where do we download these files?

Reply:
Added link for each jar files. 
Already mentioned that links in ISO8583-connector-configuration.md file also.